python/meterpreter/tests/test_ext_server_stdapi.py

```diff
@@ -320,6 +320,27 @@ def test_stdapi_sys_config_getsid(self):
         ).get("value")
         self.assertRegex(sid, "S-1-5-.*")
 
+class ExtStdNetResolveTest(ExtServerStdApiTest):
+    def test_stdapi_net_resolve_host(self):
+        # Full request from msfconsole
+        request = b'\x00\x00\x00\x0c\x00\x02\x00\x01\x00\x00\x04\x00\x00\x00\x00)\x00\x01\x00\x0264769531726942037539492283558475\x00\x00\x00\x00\x13\x00\x01\x05xrapid7.com\x00\x00\x00\x00\x0c\x00\x02\x05\xa4\x00\x00\x00\x02'
```

Comment:
This will be hard to modify in the future, and it's maybe not clear to the user were this came from

Can we use the python wrappers to build this request instead 👀

Reply:
If you wanted to TDD the rewrite of this you could do something like this as an interrim solution:

```python
python_request = ...make tlvs...
request = ...

self.assertEqual(python_request, request)
```

Obviously delete the the old `request`, and rename `python_request` to `request` once you're happy